source/hardware/controllers/native_instruments_traktor_kontrol_s3.rst

```diff
@@ -0,0 +1,254 @@
+In this mode, the sample will play while the button is held, and stop when you let go.
+
+In both modes, holding :hwlabel:`SHIFT` and pressing a button will eject the sample if it is not playing, and will rewind the sample back to the beginning if it was playing.
+
+Effects
+-------
+
+Because the S3 is an entry-level controller, the FX setup is unusual and a little complex.
+Each deck has a single effect toggle button and one knob, and on the right-hand side of the mixer there are five buttons, one for each effect chain and one for the QuickEffect.
+These buttons and knobs are used in different ways depending on how they are pushed, and together allow the DJ to customize all of the effects.
+
+There are three modes that the effect controls can be in:
+1.  The initial mode is Filter Mode.
+    This mode is indicated when the :hwlabel:`FILTER ENABLE` buttons have the same colors as the individual decks.
+    This mode is used for adjusting QuickEffects.
+1.  The next mode is Effect Chain mode.
+    This mode is indicated when the :hwlabel:`FILTER ENABLE` buttons are all the same color as one of the effect buttons.
+    This mode is used for turning individual effects in a chain on and off, and adjusting each effect's meta knob.
+1.  The last mode is Effect Focus mode.
+    This mode is indicated when :hwlabel:`FILTER ENABLE` buttons are all the same color as one of the effects, and one of the :hwlabel:`FX SELECT` buttons is blinking.
+    This mode is used for tuning individual parameters in an effect.
+
+Switching Effect Modes
+~~~~~~~~~~~~~~~~~~~~~~
+
+At any time, you can push the :hwlabel:`FILTER` :hwlabel:`FX SELECT` button to return to Filter Mode.
+If you get lost, try pusing the :hwlabel:`FILTER` button to start over.
+
+Press any :hwlabel:`FX SELECT` button to enter Effect Chain mode for that number chain.
+If you press the same :hwlabel:`FX SELECT` button again, you'll return to Filter Mode.
+Press a different :hwlabel:`FX SELECT` button to enter Effect Chain mode for that other chain.
+
+Press and hold an :hwlabel:`FX SELECT` button, then press a :hwlabel:`FILTER ENABLE` button to enter Effect Focus mode.
+The :hwlabel:`FX SELECT` button will start blinking.
+From left to right, the :hwlabel:`FILTER ENABLE` buttons will focus on the first through fourth effects in the chain.
+If you press any :hwlabel:`FX SELECT` button, you'll return to Effect Chain mode.
+
+Soft Takeover
+~~~~~~~~~~~~~
+
+The knobs have Soft Takeover mode enabled, which means you need to turn the physical knob to match the current position of the UI knob before the value will change.
+If you are wondering why it seems like the values aren't changing, you may need to rotate the knob more.
+
+Assigning Effects
+~~~~~~~~~~~~~~~~~
+
+You can assign effects to individual decks in Filter Mode.
+Press and hold :hwlabel:`FILTER ENABLE`, then press the desired :hwlabel:`FX SELECT` button or buttons.
+The :hwlabel:`FX SELECT` buttons that are bright are selected for that deck.
```

Comment:
I'm very confused by this. QuickEffects cannot have their routing changed. Did you mean to say Chain Mode?

Reply:
Also effects can't be routed individually, only chains